pkg/elasticsearch/search.go

```diff
@@ -0,0 +1,64 @@
+package elasticsearch
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/vshn/waf-tool/pkg/model"
+)
+
+// SearchUniqueID to search for a ModSecurity unique ID
+func (c client) SearchUniqueID(uniqueID string) (model.SearchResult, error) {
+
+	var buf bytes.Buffer
+	query := map[string]interface{}{
+		"query": map[string]interface{}{
+			"bool": map[string]interface{}{
+				"should": []map[string]interface{}{{
+					"match": map[string]interface{}{
+						"apache-access.uniqueID": uniqueID,
+					}}, {
+					"match": map[string]interface{}{
+						"modsec-alert.uniqueID": uniqueID,
+					}},
+				},
+			},
+		},
+	}
```

Comment:
This could be simplified for more readability with something like this:
```go
	type o map[string]interface{}
	query := o{
		"query": o{
			"apache-access.uniqueID": uniqueID,
			"bool": true,
		},
	}
```